.github/actions/do_build_ock/action.yml

```diff
@@ -4,15 +4,18 @@ description: Action to build the oneapi-construction-kit
 # Some of these are riscv or host target specific, but it does not harm at present to
 # overset cmake values
 inputs:
+  path:
+    description: 'Change to PATH before building'
+    default: '.'
   build_type:
     description: 'build type e.g Release, ReleaseAssert (default ReleaseAssert)'
     default: ReleaseAssert
   mux_targets_enable:
     description: 'mux targets to enable e.g. riscv, host (default host)'
     default: "host"
   mux_compilers_enable:
-    description: 'mux compiler target to enable e.g. riscv, host (default host)'
-    default: "host"
+    description: 'mux compiler target to enable e.g. riscv, host (no default)'
+    default: ""
```

Comment:
Harald raised the issue of the default setting for this one so comments welcome on any default value needed.

Reply:
I'm okay with the change to ""